io.edgehog.devicemanager.CellularConnectionStatus.json

```diff
@@ -0,0 +1,74 @@
+{
+  "interface_name": "io.edgehog.devicemanager.CellularConnectionStatus",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "datastream",
+  "ownership": "device",
+  "aggregation": "object",
+  "mappings": [
+    {
+      "endpoint": "/%{id}/carrier",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "Connectivity carrier operator name."
+    },
+    {
+      "endpoint": "/%{id}/cellId",
+      "type": "integer",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "The Cell ID in hexadecimal format, either 16 bit for 2G or 28 bit for 3G or 4G."
+    },
+    {
+      "endpoint": "/%{id}/homeMobileCountryCode",
```

Comment:
"The combination of MCC and MNC is called HNI (Home network identity)". So let's remove `home` from here.